### PR TITLE
Test: increase seata-core module unit test coverage

### PR DESCRIPTION
**Pull request by lightClouds917**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
#6122

### CODE REVIEW

File: core/src/test/java/io/seata/core/auth/RamSignAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * The RamSignAdapter Test
+ *
+ * @author zhongxiang.wang
```
Comment (xingfudeshi) [inline core/src/test/java/io/seata/core/auth/RamSignAdapterTest.java]: Please remove @author annotation

File: core/src/test/java/io/seata/core/compressor/CompressorTypeTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+package io.seata.core.compressor;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * The CompressorType Test
+ *
+ * @author zhongxiang.wang
```
Comment (xingfudeshi) [inline core/src/test/java/io/seata/core/compressor/CompressorTypeTest.java]: Please remove @author annotation

File: changes/en-us/2.x.md [hunk 3]
```diff
@@ -50,9 +50,9 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6147](https://github.com/apache/incubator-seata/pull/6147)] upgrade kafka-clients to 3.6.1
 
 ### test:
-- [[#6081](https://github.com/apache/incubator-seata/pull/6081)] add `test-os.yml` for testing the OS
-- [[#PR_NO](https://github.com/apache/incubator-seata/pull/PR_NO)] A brief and accurate description of PR
-- [[#6125](https://github.com/apache/incubator-seata/pull/6125)] unbind xid in TransactionTemplateTest
+- [[#6081](https://github.com/seata/seata/pull/6081)] add `test-os.yml` for testing the OS
```
Comment (slievrly) [inline changes/en-us/2.x.md]: I think need revert this line, and all seata/seata should change to apache/incubator-seata.

File: core/src/test/java/io/seata/core/auth/RamSignAdapterTest.java [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        // Use reflection to access the private method
+        Method getDateSigningKeyMethod = null;
+        try {
+            getDateSigningKeyMethod = RamSignAdapter.class.getDeclaredMethod("getDateSigningKey", String.class, String.class, String.class);
+            getDateSigningKeyMethod.setAccessible(true);
+            byte[] signingKey = (byte[]) getDateSigningKeyMethod.invoke(adapter, secret, date, signMethod);
+            Assertions.assertEquals(32, signingKey.length);
+            Assertions.assertArrayEquals(expectArray, signingKey);
+        } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
+            throw new RuntimeException(e);
```
Comment (slievrly) [inline core/src/test/java/io/seata/core/auth/RamSignAdapterTest.java]: Why is an exception caught and thrown in UT, and what does it do?

**Pull request closed without merging**